srcpkgs/aspell-it/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'aspell-it'
+pkgname=aspell-it
+version=2.2+20050523~0
```

Comment:
Debian has a newer one (with prettier version scheme) https://packages.debian.org/bullseye/aspell-it

Reply:
Fine by me, I went with what I thought was the ["official" list](https://ftp.gnu.org/gnu/aspell/dict/0index.html).

Reply:
@sgn was I supposed to resolve the conversation or were you?